includes/discovery/sensors/current/junos.inc.php

```diff
@@ -16,7 +16,7 @@
     $multiplier = 1;
     $divisor    = 1000000;
     foreach ($junos_oids as $index => $entry) {
-        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent'])) {
+        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent']) && $entry['jnxDomCurrentTxLaserBiasCurrent'] && $entry['jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold'] != 0) {
```

Comment:
That's not what I meant, here we are skipping the sensor_discovery because the low threshold is 0, we should ignore that and set it to null if it's 0 so thresholds are auto discovered